test/bridge/testhorton.py

```diff
@@ -107,7 +107,34 @@ def test_makehorton(self):
                     )
 
         elif self._hortonver == 3:
-            pass
+            # Identify attributes that should be verified
+            check = ["spinpol"]  # float or int
+            checkArr = ["atcoords", "atnums", "atcorenums"]  # one dimensional arrays
+
+            # IOData in horton 3 uses attr and initializes all possible attributes by default value of None. Thus, hasattr cannot be used here by its own.
```

Comment:
Can you reflow this comment?

Reply:
I've reflowed it for line length of 100. Thanks.